v0/destinations/persistiq/data/persistIqIdentifyConfig.json

```diff
@@ -1,9 +1,10 @@
 [
     {
         "destKey": "email",
-        "sourceKeys": "email",
-        "required": true,
-        "sourceFromGenericMap": true
+        "sourceKeys": [
```

Comment:
Shouldn't we support passing email in root also?

Reply:
like message.email ?

Reply:
Yes, as I see earlier you added the support

Reply:
it was from generic map. But there is a problem in generic mapping for email i.e. it takes the value of 0th index of externalID as last fallback value.

Reply:
for update mainly id is used , and email cannot be used